Makefile

```
@@ -11,24 +11,42 @@ help: ## Show this help message.
 		| awk 'BEGIN {FS = ":.*?## "; printf "\nUsage:\n"}; {printf "  %-15s %s\n", $$1, $$2}'
 	@echo
 
-.PHONY: publish-ami
-publish-ami: ## Build and Publish the AMI for AWS.
+.PHONY: publish-ami-ubuntu
+publish-ami-ubuntu: ## Build and Publish the AMI for AWS.
 	@cd $(AWS_DIR) && packer init .
-	@cd $(AWS_DIR) && packer build -var "ubuntu_pro_token=$(ubuntu_pro_token)" .
+	@cd $(AWS_DIR) && packer build -var "ubuntu_pro_token=$(ubuntu_pro_token)" --var-file=ubuntu.pkrvars.hcl .
 
-.PHONY: test-ami
-test-ami: fmt-ami validate-ami build-ami ## fmt, validate, and build the AMI for AWS.
+.PHONY: publish-ami-rhel
+publish-ami-rhel: ## Build and Publish the RHEL AMI for AWS.
+	@cd $(AWS_DIR) && packer init .
+	@cd $(AWS_DIR) && packer build --var-file=rhel.pkrvars.hcl .
+
+.PHONY: test-ami-ubuntu
+test-ami-ubuntu: fmt-ami validate-ami-ubuntu build-ami-ubuntu ## fmt, validate, and build the AMI for AWS.
 
-.PHONY: build-ami
-build-ami: ## Build the AMI for AWS.
+.PHONY: test-ami-rhel
+test-ami-ubuntu: fmt-ami validate-ami-rhel build-ami-rhel ## fmt, validate, and build the AMI for AWS.
```

```suggestion
test-ami-rhel: fmt-ami validate-ami-rhel build-ami-rhel ## fmt, validate, and build the AMI for AWS.
```
Noticed when running `make help`